package.json

```diff
@@ -6,21 +6,22 @@
         "build_development": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js",
         "build_production": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 NODE_ENV=production webpack --config ./webpack/webpack.config.prod.js",
         "build_test": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js --env test=true",
-        "eslint:check": "eslint . --ext .vue,.ts",
+        "eslint:check": "eslint ./arches/app/src",
+        "eslint:fix": "eslint ./arches/app/src --fix",
         "eslint:watch": "nodemon --watch . --ext ts,vue --exec yarn --silent eslint:check",
-        "eslint:fix": "eslint . --ext .vue,.ts --fix",
         "gettext:extract": "vue-gettext-extract",
         "gettext:compile": "vue-gettext-compile",
         "ts:check": "vue-tsc --noEmit",
         "ts:watch": "vue-tsc --watch --noEmit",
-        "start": "cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack serve --config ./webpack/webpack.config.dev.js"
+        "start": "cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack serve --config ./webpack/webpack.config.dev.js",
+        "vitest": "vitest --run --coverage"
     },
     "repository": {
         "url": "git@github.com:archesproject/arches.git",
         "type": "git"
     },
     "devDependencies": {
-        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#dev/7.6.x"
+        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#cbyrd-upgrade-dev-dependencies"
```

Comment:
This needs to be changed && yarn.lock rebuilt after merging https://github.com/archesproject/arches-dev-dependencies/pull/17

Reply:
done 👍